### ISSUE TITLE
Alguna duda! come on!

### ISSUE THREAD

**Issue by SihamTabik** [SihamTabik/Foro_Grupo-A4_Inform-tica_I#1]

**Comment by makixun** [comment]
Hola! No tenemos que entregar los ejercicios de la primera práctica?

**Comment by SihamTabik** [comment]
Sí, hay que entregar un zip de todos los códigos de la primera práctica.

**Comment by makixun** [comment]
Buenas tardes! Podría poner ya en Decsai el guión 2? Yo ya acabé el 0 y el 1.
Saludos.

**Comment by SihamTabik** [comment]
Sí, perfecto!
Saludos

**Comment by makixun** [comment]
En el ejercicio 7 de la progresión olvidé hacer la suma de los términos, la dejo por aqui.
int i = valor1;
int suma = 0;
while(i != valor_iesimo*razon){
	suma = suma + i;
	i = i * razon;
}

cout << suma;

PD: Con lo del guión 2 me refería a si usted podía subir a decsai los ejercicios que hay que hacer del guión, no a si podia subirlos yo ya hechos, obviamente no los tengo. Por si habia confusión o malinterpretación :)

Saludos.

**Comment by makixun** [comment]
Gracias!!

**Comment by makixun** [comment]
Hola! Podría abrir la entrega en decsai? Que ninguno hemos podido entregar nada todavía al estar cerrada. Saludos.

**Comment by SihamTabik** [comment]
Hola Miguel,

Estoy teniendo problemas con mi cuenta en DECSAI. Comentales a tus compañeros que me envien la practica por email.

Saludos
Siham

**Comment by SihamTabik** [comment]
Hola Miguel,
Puedes probar subir tu entrega. Confirmame que todo funciona.
Gracias y saludos
Siham

**Comment by makixun** [comment]
Funciona perfectamente! Saludos.

**Comment by makixun** [comment]
Podría usted ya de camino poner el guión 3 disponible?

**Comment by SihamTabik** [comment]
Ya está visible.

**Comment by makixun** [comment]
Ha puesto cinco entregas, tres donde pone "Entrega guion x" y dos de "Práctica x". ¿Donde se lo entregamos?

**Comment by SihamTabik** [comment]
Ya sólo hay:
Entrega guion 0
Entrega guion 1
Entrega guion 2
Entrega guion 3

**Comment by makixun** [comment]
No termino de entender que hay que hacer en el ejercicio 6 del guión 3. ¿A que se refiere exactamente?
Saludos.

**Comment by SihamTabik** [comment]
Hola Miguel,

Supongo que te refieres a este ejercicio:
6. Implementa un programa que dada una matriz 3x3, normalice todos los elementos
por el mayor de la matriz. Después deberá comprobar si la suma de los elementos de
las dos diagonales coinciden.

Yo interpreto lo siguiente:
Dada una matriz:
               a b c
MATRIZ= d e f 
               g h i

maximum=max(a,b,c,d,e,f,g,f,i)
               a/maximum b/maximum c/maximum
MATRIZ= d/maximum e/maximum f/maximum 
               g/maximum h/maximum i/maximum
Comprobar que: a/maximum+e/maximum+i/maximum = c/maximum+e/maximum+g/maximum

**Comment by SihamTabik** [comment]
Hola Miguel,

Supongo que te refieres a este ejercicio:
6. Implementa un programa que dada una matriz 3x3, normalice todos los elementos
por el mayor de la matriz. Después deberá comprobar si la suma de los elementos de
las dos diagonales coinciden.

Yo interpreto lo siguiente:
Dada una matriz:
a b c
d e f
g h i

maximum=max(a,b,c,d,e,f,g,f,i)
matriz se convierte en:
a/maximum b/maximum c/maximum
d/maximum e/maximum f/maximum
g/maximum h/maximum i/maximum
Comprobar que: a/maximum+e/maximum+i/maximum = c/maximum+e/maximum+g/maximum

**Comment by makixun** [comment]
Vale, gracias!!

**Comment by juanjojarque99** [comment]
Hola Siham soy Juanjo , hemos estado hablando y muchos no nos enteramos del guion 3 mucho lo podríamos ir haciendo mañana en clase todos juntos?

**Comment by SihamTabik** [comment]
Hola Juanjo,

De acuerdo, mañana lo hacemos juntos en clase.

Saludos
Siham

**Comment by makixun** [comment]
Hola, mañana no podré asistir a las prácticas de por la tarde. Tengo ya el guión 4 terminado. Podría no restarme puntos de asistencia mañana? 

Saludos.

**Comment by SihamTabik** [comment]
Hola Miguel,

Acabo de crear en DECSAI la entrega de la Práctica 4 . Así que ya puedes realizar tu entrega.
En cuanto a tu pregunta, no te preocupes, no te voy a restar puntos.
Saludos,
